client/src/components/useContestChoiceNameStandardizations.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  useMutation,
+  UseMutationResult,
+  useQuery,
+  useQueryClient,
+  UseQueryResult,
+} from 'react-query'
+
+import { ApiError, fetchApi } from '../utils/api'
+import { contestsQueryKey } from './useContests'
+
+export interface IContestChoiceNameStandardizations {
+  [jurisdictionId: string]: {
+    [contestId: string]: {
+      [cvrChoiceName: string]: string | null // Standardized choice name
+    }
+  }
+}
+
+export interface IContestChoiceNameStandardizationsResponse {
+  standardizations: IContestChoiceNameStandardizations
+}
+
+export const contestChoiceNameStandardizationsQueryKey = (
+  electionId: string
+): string[] => ['election', electionId, 'contestChoiceNameStandardizations']
+
+export const useContestChoiceNameStandardizations = (
+  electionId: string
+): UseQueryResult<IContestChoiceNameStandardizationsResponse, ApiError> =>
+  useQuery<IContestChoiceNameStandardizationsResponse, ApiError>(
+    contestChoiceNameStandardizationsQueryKey(electionId),
+    () => {
+      return fetchApi(
+        `/api/election/${electionId}/contest/choice-name-standardizations`
+      )
+    }
+  )
+
+export const useUpdateContestChoiceNameStandardizations = (
+  electionId: string
+): UseMutationResult<
+  IContestChoiceNameStandardizations,
+  ApiError,
+  IContestChoiceNameStandardizations
+> => {
+  const queryClient = useQueryClient()
+
+  const putStandardizations = (
+    newStandardizations: IContestChoiceNameStandardizations
+  ) => {
+    return fetchApi(
+      `/api/election/${electionId}/contest/choice-name-standardizations`,
+      {
+        method: 'PUT',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(newStandardizations),
+      }
+    )
+  }
+
+  return useMutation(putStandardizations, {
+    onSuccess: () => {
+      queryClient.invalidateQueries(
+        contestChoiceNameStandardizationsQueryKey(electionId)
+      )
+      queryClient.invalidateQueries(contestsQueryKey(electionId))
```

Comment:
Invalidating the contests query is necessary because it's used to populate this table on the audit setup review screen. I add this invalidation in a few places in this commit.

<img alt="table" src="https://github.com/votingworks/arlo/assets/12616928/0a3d1beb-5aec-43f3-a481-a659f3a64bf7" width=300 />